sdks/java/expansion-service/src/main/java/org/apache/beam/sdk/expansion/service/ExpansionService.java

```diff
@@ -137,402 +132,365 @@ public interface ExpansionServiceRegistrar {
   public static class ExternalTransformRegistrarLoader
       implements ExpansionService.ExpansionServiceRegistrar {
 
-    private static final SchemaRegistry SCHEMA_REGISTRY = SchemaRegistry.createDefault();
-
     @Override
-    public Map<String, ExpansionService.TransformProvider> knownTransforms() {
-      ImmutableMap.Builder<String, ExpansionService.TransformProvider> builder =
```

Comment:
I was getting duplicate key errors from this builder, which is not built until after everything. So this is actually the thing that specifically I needed to refactor to get more messages in the middle. (and I did figure out it was a splittable keyed pardo registration or something like that)

Reply:
Ack. Didn't go through all this logic. Assume this is just a refactoring.